client/src/components/Navbar.js

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+import logoImage from './../images'
+import {
+  Collapse,
+  Navbar,
+  NavbarToggler,
+  NavbarBrand,
+  Nav,
+  NavItem,
+  NavLink,
+  Dropdown,
+  DropdownToggle,
+  DropdownMenu,
+  DropdownItem, Button } from 'reactstrap';
+
```

Comment:
This file is pretty big, would you be able to add in a comment here? The format will be something like this! Ideally we'll come back in the future and refactor this code to make it easier to read! Thank you :) 

```
/**
 * TODO: Make files better organized
*/
```